src/Test/Utilities/DiagnosticAnalyzerTests.Extensions.cs

```diff
@@ -13,7 +13,12 @@ namespace Microsoft.CodeAnalysis.UnitTests
 {
     public static class DiagnosticAnalyzerTestsExtensions
     {
-        public static void Verify(this IEnumerable<Diagnostic> actualResults, DiagnosticAnalyzer analyzer, params DiagnosticResult[] expectedResults)
+        public static void Verify(
+            this IEnumerable<Diagnostic> actualResults,
+            DiagnosticAnalyzer analyzer,
+            bool printActualDiagnosticsOnFailure,
+            string expectedDiagnosticsAssertionTemplate,
+            params DiagnosticResult[] expectedResults)
```

Comment:
Yes, there are overloads that doesn't need an additional parameters. Unfortunately, the git diff is not very helpful is showing that.